packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5ChangePassword.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react';
+import { useMT5AccountHandler, useQueryParams } from '@/hooks';
+import { validPassword } from '@/utils';
+import { useTradingPlatformPasswordChange } from '@deriv/api-v2';
+import { Button, Modal, PasswordInput, Text } from '@deriv-com/ui';
+import { CFDPlatforms } from '../../constants';
+
+const MT5ChangePassword = () => {
+    const { isLoading: tradingPlatformPasswordChangeLoading, mutateAsync: tradingPasswordChange } =
+        useTradingPlatformPasswordChange();
+    const { handleSubmit: createMT5Account, createMT5AccountStatus } = useMT5AccountHandler();
+    const { openModal } = useQueryParams();
+
+    const [currentPassword, setCurrentPassword] = useState('');
+    const [newPassword, setNewPassword] = useState('');
+
+    const changePasswordAndCreateAccountHandler = async () => {
+        await tradingPasswordChange({
+            new_password: newPassword,
+            old_password: currentPassword,
+            platform: CFDPlatforms.MT5,
+        }).then(() => createMT5Account(newPassword));
+    };
+
+    useEffect(() => {
+        if (createMT5AccountStatus === 'success') openModal('MT5SuccessModal');
+    }, [createMT5AccountStatus, openModal]);
+
+    const passwordValidationChecklist = [
+        '8 to 16 characters',
+        'A special character such as ( _ @ ? ! / # )',
+        'An uppercase letter',
+        'A lowercase letter',
+        'A number',
+    ];
+
+    return (
+        <React.Fragment>
+            <Modal.Body className='lg:p-24 lg:w-[525px]'>
```

Comment:
Do we need this width ?

Reply:
yes, to contain the modal from stretching out

Reply:
then lets add max-width instead of deez

Reply:
w-max:
<img width="1436" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/09864cd8-7b51-4752-8603-535739decfed">

Reply:
Huh